src/main/java/frc/utils/calibration/sysid/SysIdObject.java

```diff
@@ -0,0 +1,134 @@
+package frc.utils.calibration.sysid;
+
+import com.ctre.phoenix6.SignalLogger;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.utils.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.function.Consumer;
+
+import static edu.wpi.first.units.Units.Seconds;
+import static edu.wpi.first.units.Units.Volts;
+
+public class SysIdObject {
```

Comment:
I don't think `SysIdObject` is a good name, consider changing to something like `SysIdCalibrator`